models/users.py

```diff
@@ -6,4 +6,5 @@ class Users(BaseModel):
     server_id = CharField()
 
     class Meta:
+        primary_key = CompositeKey('discord_id', 'server_id')
```

Comment:
Weren't there issues with have a foreign composite key? I added the UNIQUE constraints one line below which should work good enough for our case